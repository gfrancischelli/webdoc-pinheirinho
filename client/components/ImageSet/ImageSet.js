import React from 'react';
import breakpoints from 'media-query.js';

export default ({ url }) => (
  <picture>
    <source
      media={`max-width: ${breakpoints.phone}`}
      srcSet={ `${ url }?width=${ 370 }` } />
    <source
      media={`max-width: ${breakpoints.tablet}`}
      srcSet={ `${ url }?width=${ 400 }` } />
    <img 
      src={ url }
      className='o-card__figure' /> 
  </picture>
)

