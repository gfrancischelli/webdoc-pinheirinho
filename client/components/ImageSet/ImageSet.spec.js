import React from 'react';
import ImageSet from './ImageSet';

it('should render correctly', () => {
  const wrapper = shallow(
    <ImageSet url=''>
      <img />
    </ImageSet>
  )
  expect(wrapper).toMatchSnapshot();
})
