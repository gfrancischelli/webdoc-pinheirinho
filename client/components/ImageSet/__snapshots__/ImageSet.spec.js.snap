exports[`test should render correctly 1`] = `
<picture>
  <source
    media="max-width: 420px"
    srcSet="?width=370" />
  <source
    media="max-width: 768px"
    srcSet="?width=400" />
  <img />
</picture>
`;
